src/components/views/elements/Dropdown.js

```diff
@@ -115,7 +115,7 @@ export default class Dropdown extends React.Component {
 
     componentWillReceiveProps(nextProps) {
         this._reindexChildren(nextProps.children);
-        const firstChild = React.Children.toArray(nextProps.children)[0];
+        const firstChild = nextProps.children[0];
```

Comment:
If you use `React.children` you get the full key of the child, something like `.$GB`. I should've mentioned this tbf..